scarb/tests/build_cairo_plugin.rs

```diff
@@ -530,7 +530,50 @@ fn can_return_aux_data_from_plugin() {
             [..]Compiling hello v1.0.0 ([..]Scarb.toml)
             [SomeMacroDataFormat { msg: "Hello from some macro!" }]
             [..]Finished release target(s) in [..]
-            [..]Running hello
-            [..]Run completed successfully, returning [..]
+        "#});
+}
+
+#[test]
+fn can_read_token_stream_metadata() {
+    let temp = TempDir::new().unwrap();
+    let t = temp.child("some");
+    simple_project_with_code(
+        &t,
+        indoc! {r##"
+        use cairo_lang_macro::{ProcMacroResult, TokenStream, attribute_macro};
+
+        #[attribute_macro]
+        pub fn some_macro(token_stream: TokenStream) -> ProcMacroResult {
+            println!("{:?}", token_stream.metadata());
```

Comment:
```suggestion
            println!("{:#?}", token_stream.metadata());
```

and you'll get much easier to diff output